File: cfgs/java_cfgs_foo/MINIMUM_XOR_VALUE_PAIR.java
Nodes
1;special;;;
2;statement;int min_xor = Integer.MAX_VALUE;14;
3;statement;int i = 0;15;
4;conditional;foo_1 < foo_2;16;
5;statement;int j = foo_1 + 1;17;
6;conditional;foo_1 < foo_2;18;
7;statement;min_xor = fun_1(foo_1, foo_2[foo_3] ^ foo_4[foo_5]);19;
8;statement;j++;19;
9;statement;i++;17;
10;statement;return foo_1;20;
11;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;10;
4;6;
5;7;9;
6;8;
7;6;
8;4;
9;11;
10;

File: cfgs/pythoncfgs/MINIMUM_XOR_VALUE_PAIR.py
Nodes
1;statement;fun_1();8;
2;statement;foo_1 = 999999;9;
3;statement;foo_1 = 0;10;
4;conditional;foo < foo;11;
5;statement;return foo_1;15;
6;conditional;foo < foo;12;
7;statement;foo_1 = foo_2[foo_3] ^ foo_4[foo_5];13;
8;statement;foo_1 = fun_1(foo_2, foo_3);14;
9;exit;;
Edges
0;2;
1;3;
2;4;
3;5;6;
4;9;
5;4;7;
6;8;
7;6;
8;
